tox.ini

```diff
@@ -62,7 +65,10 @@ commands =
 
 [testenv:tests-yelpy]
 envdir = .tox/py38-linux/
+setenv =
+    PIP_INDEX_URL = http://169.254.255.254:20641/simple/
```

Comment:
we should probably refactor our env handling at some point so that this isn't required here, but that's probably not something we wanna tackle in this PR